service-base/src/main/java/org/eclipse/hono/service/management/device/DeviceManagementHttpEndpoint.java

```diff
@@ -37,6 +37,7 @@
  * Registration API</a>. It receives HTTP requests representing operation invocations and sends them to an address on
  * the vertx event bus for processing. The outcome is then returned to the peer in the HTTP response.
  */
+@Deprecated(forRemoval = true)
```

Comment:
maybe you can also add a `@deprecated` tag to the JavaDoc explaining why this class will be removed?